### DISCUSSION TITLE
php5.3: Call to undefined function Sentry\init()

### DISCUSSION THREAD

**Discussion by pixeline** [getsentry/sentry-php#1302] [Q&A] [answered]
I would like to use Sentry on a php5.3 application (stuck there for a while, until i finish porting the code to PHP8+). I get this error message:  

> Fatal error: Call to undefined function Sentry\init() in /var/www/html/membres/test_sentry.php on line 9

Line 8 and 9: 
```php
require '../vendor/autoload.php';
\Sentry\init(array('dsn' => 'https://xxxx.ingest.sentry.io/xxxxx' ));
```

Here is my composer.json

```json
{
    "require": {
        "sentry/sentry": "^1.0"
    },
    "config": {
        "platform": {
            "php": "5.3"
        }
    }
}
```

Does this mean Sentry cannot be run on php5.3?

**Comment by Jean85** [comment, marked as answer]
Sentry 1.x works with PHP 5.3. Your mistake is using the docs, which refers to the 3.x series, which is profoundly different.

You can look around in the 1.x branch, and in the readme there to start over: https://github.com/getsentry/sentry-php/tree/1.x#usage

**Reply by pixeline** [reply]
ouch, thank you for the heads up!
